### DISCUSSION TITLE
Dubbi creazione fattura elettronica con campi PA_ che non trovo più

### DISCUSSION THREAD

**Discussion by ilTato** [fattureincloud/api#114] [Q&A] [answered]
Buongiorno a tutti,

sono alla prese con la creazione della fattura elettronica utilizzando la nuova versione delle API v.2 (utilizzando SDK),
a suo tempo nella versione v.1 c'erano una serie di campi che iniziavano con PA_ che qui non riesco a trovare, 
se qualcuno può darmi una indicazione su dove trovarli e ovviamente dove inserirli in modo da creare una fattura elettronica in modo corretto e non rischiare che mi torni indietro.

Per esempio:

PA = valore 'true', non ho trovato il campo
PA_tipo_cliente = inserivo il valore 'B2B' (non ho trovato questo campo)
PA_codice = inserivo il 'codice univoco' del cliente (se presente), ma non ho trovato il campo
PA_pec = inserivo l'eventuale 'indirizzo PEC' al posto del codice univoco (in realtà ho trovato l'indirizzo PEC ma è riferito all'anagrafica, non vorrei sbagliare campo), visto che su questo nuovo tracciato record v.2 non l'ho trovato.
PA_istituto_credito = lasciavo 'vuoto'
PA_iban = inserivo il mio iban fisso
PA_beneficiario = inserivo la mia ragione sociale

METODO_PAGAMENTO = inserivo una breve descrizione ...che appariva sulla fattura, qui invece sembra tutto agganciarsi alla tabella (vorrei solo avere conferma)
METODO_TITOLO1 = anche in questo caso inserivo una brave descrizione che appariva sulla fattura cartacea (vedi il campo precedente)
METODO_DESC1 = idem come il campo sopra

infine, volutamente non inserivo mai il prezzo 'netto' nel dettaglio delle righe della fattura perchè impostavo (se ricordo bene)
un campo chiamato 'prezzi_ivati' con il valore 'true' in questo modo ricalcolava sempre tutti gli importi, altrimenti mi ricordo di avere sempre piccoli problemi sull'arrotondamento.

Mi auguro tanto che qualcuno possa darmi una dritta in modo da poter finalmente terminare questa transizione dalla versione 1 alla versione 2 delle API (augurandomi che non esista in futuro la versione 3).

Un cordiale saluto a tutti
ilTato

**Comment by ilTato** [comment, marked as answer]
Buongiorno, ho trovato altri post dove vengono menzionati gli stessi campi che sono alla ricerca anche io.
Alcuni di quei campi li ho trovati e capito che devo anche impostare eiinvoice = true.
Sono che non sono riuscito a trovare (per ora) l'equivalente del campo prezzi_ivati, sempre che esista.
Invece per la forma di pagamento ci sto provando con alcuni esempi ....appena avrò delle notizie vi aggiorno.
Buona giornata a tutti
ilTato

**Reply by emajo** [reply]
Ciao @ilTato il campo che corrisponde al vecchio `prezzi_ivati` nelle api v2 è `use_gross_prices`.

**Reply by ilTato** [reply]
Buongiorno @emajo ,
grazie per l'aiuto non lo avevo proprio notato, ho già provato la cosa è funziona correttamente,
un ringraziamento speciale a tutti voi che ci aiutate in caso di bisogno ed al tempo che ci dedicate,
veramente grazie!
ilTato
